Cython/Compiler/Builtin.py

```diff
@@ -279,8 +279,7 @@ def declare_in_type(self, self_type):
 # resulting in unintuitive runtime behavior and segfaults.
 #    ("bool",   "&PyBool_Type",     []),
 
-    ("int",     "&PyInt_Type",     []),
-    ("long",    "&PyLong_Type",    []),
```

Comment:
Do we want to remove `long` or just keep it as an alias for `int`?

Reply:
Hmm. Use cases? I only see `isinstance(x, (int, long))` in code that wanted to support Python 2.x. Everything else probably meant C `long` instead. And code that wanted to keep supporting Python 2 would probably be better off with a compilation failure when migrating to Cython 3.1 instead of mapping it to Python 'int'.

Reply:
I don't really have a use-case in mind. There's probably some old code around that this change will break, but it's an easy enough fix anyway so it's probably OK

Reply:
I kept the name `long` available in language level 2, mapping it directly to the `int` type.